Assets/MixedRealityToolkit.Examples/Demos/EyeTracking/Demo_ScrollPanZoom/Scripts/BaseClasses/PanZoomBase.cs

```diff
@@ -121,7 +121,7 @@ public abstract class PanZoomBase : MonoBehaviour,
         private Vector3 navPos = Vector3.zero;
         private bool isFocused = false;
         internal bool isZooming = false;
-        public bool handZoomEnabledOnStartup = false;
+        internal bool ZoomGestureEnabledOnStartup = false;
```

Comment:
curious as to why this changed to internal

Reply:
The PanZoomBase class is consumed both by the scroll as well as the pan-and-zoom scripts. The scroll scripts don't use zoom though and hence it doesn't make sense to show this parameter in the Unity Editor.